src/Scripts/EventBus.gd

```diff
@@ -0,0 +1,3 @@
+extends Node
+
+signal move_entity(entity, x, y)
```

Comment:
This is a very strange way to use the event bus, but very cool - no code required! 👍